src/modm/platform/random/stm32/random_number_generator.hpp.in

```diff
@@ -37,6 +38,8 @@ public:
 	static inline void
 	enable()
 	{
+		Rcc::enable<Peripheral::Rng>();
+		Rcc::reset<Peripheral::Rng>();
```

Comment:
Yeah, remember how the RNG never worked? HAVE YOU TRIED TURNING IT ~~OFF AND~~ ON ~~AGAIN~~???